generate.go

```diff
@@ -1,5 +1,7 @@
 // +build gen
 
+//go:generate go test ../glow/
```

Comment:
Do we need testing at go-generate?

Reply:
Not necessarily, but it does make things easier to have tests run automatically when doing the `edit glow -> go generate -> check diffs -> repeat` loop, without much drawbacks.

Reply:
I feel like the test should be separately done e.g. in GitHub Actions, but I'd like to let the other maintainers decide this.

Reply:
My preference is to use generate directives only to generate code. The developer should do `go generate && go test` to both generate and run tests.